src/parser/model/cloudformation-resource.ts

```diff
@@ -30,12 +30,6 @@ export class CloudFormationResource extends Resource {
         if (this.bindings) {
             this.includeMasterAccount = this.bindings.IncludeMasterAccount;
 
-            if (Array.isArray(this.bindings.ExcludeAccounts)) {
-                this.excludeAccounts = this.bindings.ExcludeAccounts as Array<Reference<AccountResource>>;
-            } else {
-                this.excludeAccounts = [ this.bindings.ExcludeAccounts as Reference<AccountResource> ];
-            }
-
```

Comment:
This piece was to initialize the excludeAccounts property similar to how the `this.regions` was being initialized. Not sure if that was needed but it seemed like a good idea.